tests/kuttl/tests/deploy/10-assert-deploy-ironic.yaml

```diff
@@ -141,7 +141,7 @@ apiVersion: ironic.openstack.org/v1beta1
 kind: IronicNeutronAgent
 metadata:
   finalizers:
-  - IronicNeutronAgent
+  - openstack.org/ironicneutronagent
```

Comment:
Was this changed in the Golang code as well?  I'm having trouble seeing it

Reply:
not go code change is required. its the default finalizer it gets from lib-common https://github.com/openstack-k8s-operators/ironic-operator/blob/main/controllers/ironicneutronagent_controller.go#L196